webpack/ForemanInventoryUpload/Components/PageHeader/components/ToolbarButtons/toolbarButtons.scss

```diff
@@ -8,4 +8,8 @@
   .sync_button .spinner {
     margin: 3px 5px 0 -3px;
   }
+
+  .cloud-connector-pending-button {
+    display: inline-block;
```

Comment:
![Screenshot (40)](https://user-images.githubusercontent.com/26363699/107150961-eede6680-6968-11eb-80bd-3dadd52eaec4.png)

Button jumped above the other button now :man_dancing: